benchmark/single.go

```diff
@@ -93,23 +93,27 @@ func SingleFindOneByID(ctx context.Context, tm TimerManager, iters int) error {
 		return err
 	}
 	coll := db.Collection("corpus")
+
+	ids := make([]interface{}, iters)
 	for i := 0; i < iters; i++ {
-		id := int32(i)
-		res, err := coll.InsertOne(ctx, doc.Set("_id", bsonx.Int32(id)))
+		res, err := coll.InsertOne(ctx, doc)
```

Comment:
Suggest reverting to using the `Int32` ID. This benchmark is defined https://github.com/mongodb/specifications/blob/master/source/benchmarking/benchmarking.rst#find-one-by-id

> using sequential _id values. (1 to 10,000)

Having the driver generate an ObjectID `_id` may slightly alter the results.

Reply:
Good catch! Will update.